src/cpu/aarch64/jit_uni_dw_convolution.hpp

```diff
@@ -96,9 +96,11 @@ struct jit_uni_dw_convolution_fwd_t : public primitive_t {
 
     std::unique_ptr<jit_uni_dw_conv_fwd_kernel<isa, src_type>> kernel_;
 };
-
+// To remove after resolving dependency on 1x1
```

Comment:
I don't understand this comment, sorry. Could you elaborate please?

Reply:
Hi @jondea earlier this variable is used to call template code in dispatch list for the kernel. But now I have removed this variable from the code from cpu_convolution_list. But same variable template is still used to call the DW kernel in JIT_1x1_kernel files. We will remove this in future release.